src/main/java/ch/njol/skript/effects/EffScreaming.java

```diff
@@ -0,0 +1,65 @@
+package ch.njol.skript.effects;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.*;
+import ch.njol.skript.lang.Effect;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.util.Kleenean;
+import org.bukkit.entity.Enderman;
+import org.bukkit.entity.Goat;
+import org.bukkit.entity.LivingEntity;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+
+@Name("Make Entity Scream")
+@Description("Make a goat or enderman (Paper) start or stop screaming.")
+@Examples({
+	"make last spawned goat start screaming",
+	"force last spawned goat to not scream",
+	"",
+	"make {_enderman} scream",
+	"force {_enderman} to stop screaming"
+})
+@RequiredPlugins("Paper (enderman)")
+@Since("INSERT VERSION")
+public class EffScreaming extends Effect {
+
+	private static final boolean SUPPORTS_ENDERMAN = Skript.methodExists(Enderman.class, "setScreaming", boolean.class);
+
+	static {
+		Skript.registerEffect(EffScreaming.class,
+			"make %livingentities% (start screaming|scream)",
+			"force %livingentities% to (start screaming|scream)",
+			"make %livingentities% (stop screaming|not scream)",
+			"force %livingentities% to (stop screaming|not scream)");
```

Comment:
```suggestion
			"make %livingentities% stop screaming",
			"force %livingentities% to stop screaming");
```
this is by far the most schizophrenic syntax in skript